test/get_annotated_tasks_test.dart

```diff
@@ -0,0 +1,69 @@
+// Copyright 2013 Google. All rights reserved. Use of this source code is
```

Comment:
And generally we don't update the copyright year when modifying a file - we just need to make sure that new files have the copyright (and that we update the year for that file).